src/sections/Community/Handbook/repository.js

```diff
@@ -208,7 +208,7 @@ const Repository = () => {
                       </tr>
                     </thead>
                     {backendProject.subdata.map((subdata) => {
-                      const { project,image,language,description,repository,link,maintainers_name } = subdata;
+                      const { project,image,language,description,repository,link,maintainers_name,accessRequired } = subdata;
```

Comment:
@H-A-R-S-H-K, why these changes?

Reply:
@Ashparshp In the frontend projects subdata this type of code was followed so i thought to make it same for backend.

Reply:
is this was the ask @H-A-R-S-H-K ?

Reply:
@sudhanshutech well we have to show the status of SMI Conformance repo as archived. To achieve that I added a key "accessRequired" with value as "*archived" in the backend projects subdata and added a div to display that "*archived".

<img width="783" alt="Screenshot 2024-08-24 at 2 33 43 PM" src="https://github.com/user-attachments/assets/7cd0df7b-68ea-4f57-aa27-353621fd69cc">

was this not the ask ?